cmd/list/region/cmd.go

```diff
@@ -127,6 +127,9 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(0)
 	}
 
+	if !hypershiftEnabled && cmd.Flags().Changed("hosted-cp") {
+		r.Reporter.Warnf("'%s' not set for current organization", ocm.HypershiftCapability)
+	}
```

Comment:
Can this check be moved above to right after `IsCapabilityEnabled` is called? Otherwise we're doing a bunch of region support checks to just error out in the end.

Reply:
so far it was just a warning and not exiting, so it was good to have it there.
Changed to error with exit and moved up.

Reply:
Yes, I think this makes sense - otherwise the warning might just confuse users.